semanticdb/metac/src/main/scala/scala/meta/cli/Metac.scala

```diff
@@ -0,0 +1,33 @@
+package scala.meta.cli
+
+import java.io._
+import java.net._
+import java.nio.channels._
+import java.nio.file._
+import scala.meta.internal.semanticdb.scalac._
+import scala.tools.nsc.{Main => ScalacMain}
+
+object Metac {
+  def main(args: Array[String]): Unit = {
+    sys.exit(process(args))
+  }
+
+  def process(args: Array[String]): Int = {
+    val manifestDir = Files.createTempDirectory("semanticdb-scalac_")
+    val resourceUrl = classOf[SemanticdbPlugin].getResource("/scalac-plugin.xml")
+    val resourceChannel = Channels.newChannel(resourceUrl.openStream())
+    val manifestStream = new FileOutputStream(manifestDir.resolve("scalac-plugin.xml").toFile)
+    manifestStream.getChannel().transferFrom(resourceChannel, 0, Long.MaxValue)
+    manifestStream.close()
+    val pluginClasspath = classOf[SemanticdbPlugin].getClassLoader match {
+      case null => manifestDir.toString
+      case cl: URLClassLoader => cl.getURLs.map(_.getFile).mkString(File.pathSeparator)
```

Comment:
Does this work for coursier bootstrap scripts? In scalafix you implemented something custom https://github.com/scalacenter/scalafix/blob/325ad576a7c4c5e8a5286195a1a83d091c77ac2d/scalafix-reflect/src/main/scala/scalafix/internal/reflect/ScalafixToolbox.scala#L74 Any chance we can reuse?

Reply:
Let's look into this separately. I added an entry to my issue tracker.